package.json

```diff
@@ -27,15 +27,15 @@
     "build": "tsc -p tsconfig.dist.json --declaration && cp README.md dist/README.md && cp LICENSE dist/LICENSE"
   },
   "dependencies": {
-    "@jupiterone/integration-sdk-cli": "^8.1.1",
-    "@jupiterone/integration-sdk-dev-tools": "^8.1.1",
-    "ajv": "^8.8.2",
+    "@jupiterone/integration-sdk-cli": "^8.2.0",
+    "@jupiterone/integration-sdk-core": "^8.2.0",
+    "ajv": "^8.9.0",
     "js-yaml": "^4.1.0",
     "nodegit": "^0.27.0",
     "ts-node": "^10.4.0"
   },
   "devDependencies": {
-    "@jupiterone/typescript-tools": "^15.0.0",
```

Comment:
Having both `@jupiterone/integration-sdk-dev-tools` and `@jupiterone/typescript-tools` installed was causing Jest to have colliding versions.